hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/VolumeIOStats.java

```diff
@@ -156,6 +157,8 @@ public long getReadTime() {
   public long getWriteTime() {
     return writeTime.value();
   }
-
-
+  @Metric(value = "Returns the storage directory name for the volume")
```

Comment:
I think we should conform with the current style of annotation usage, and annotate the `private String storageDirectory` field instead of the method here, with that I would like to ask you to consider either to skip the description, or add a description to other fields as well, would these descriptions tell more about the metric that the name does not tell, or does it help understand the metrics on the UI? If yes, then let's add description, if not, let's skip I think.

Also a small thing, but we should keep the convention of leaving an empty line before a new method declaration.

Reply:
There is a bug with @Metric annotations for fields, which works properly for methods. Turns out that @Metric on fields ignores type. This needs to be fixed on hadoop-common side and we either use all annotations on methods or this hybrid way stays.

Fixed the missing empty line and removed the unnecessary description.

Reply:
> Turns out that `@Metric` on fields ignores type.

Maybe I'm missing something, but we don't have `type` specified here, so what is being ignored?

(BTW, please write `@Metric` as code, to avoid mentioning the GitHub user named [Metric](https://github.com/Metric).)

Reply:
> because it gets converted automatically to it for methods like this.

Oh, then I was missing this.  Thanks for the info.